File: pythonproblems/day11/rev-str.py
# reverse a array string based on length in assending order
arr=["hi","heelo","hlo","hoya","buddy"]

for i in range(len(arr)):
    for j in range(len(arr)):
        if len(arr[i])<len(arr[j]):
            temp=arr[i]
            arr[i]=arr[j]
            arr[j]=temp
print(arr)

# reverse a string in desecding order

arr1=["hi","heelo","hlo","hoya","buddy"]
for k in range(len(arr1)):
    for h in range(len(arr1)):
        if len(arr1[k])>len(arr1[h]):
            temp=arr1[k]
            arr1[k]=arr1[h]
            arr1[h]=temp
print(arr1)

File: jsprobelms/day11/rev-str.js
// reverse a string in ascending order
arr=["hii","hello","hi","hoya","buddy"]
for(i=0;i<arr.length;i++){
    for(j=0;j<arr.length;j++){
        if(arr[i].length<arr[j].length){
            temp=arr[i]
            arr[i]=arr[j]
            arr[j]=temp
        }

    }
}
console.log(arr);

// reverse a string in desending order
arr=["hii","hello","hi","hoya","buddy"]
for (k=0;k<arr.length;k++){
    for(h=0;h<arr.length;h++){
        if(arr[k].length>arr[h].length){
            temp=arr[k]
            arr[k]=arr[h]
            arr[h]=temp
        }
    }
}
console.log(arr);
